File: Data Structures & Algorithms/count-the-number-of-consistent-strings/submission-0.cs
public class Solution {
    public int CountConsistentStrings(string allowed, string[] words) {

        int[] code = new int[26];

        int count = 0;

        foreach(char c in allowed){
            code[c - 'a']=1;
        }   

        foreach(string s in words){
            bool consistent = true;

                foreach(char c in s){
                    if( code[c-'a'] == 0){
                        consistent = false;
                    }
                }    
            if(consistent){
                count++;
            }

        }
        
        return count;


    }
}

File: Data Structures & Algorithms/find-words-that-can-be-formed-by-characters/submission-0.py
class Solution:
    def countCharacters(self, words: List[str], chars: str) -> int:
        count = 0
       

        for s in words:
            valid = True
            code = [0]*26
            for c in s:
                code[ord(c) - ord('a')] +=1
        
            for c in chars:
                code[ord(c) - ord('a')] -=1

            for val in code:
                if(val > 0):
                    valid = False
            
            if valid:
                count+=len(s)
                valid = True

        return count
        